spring-data-cassandra/src/main/resources/org/springframework/data/cassandra/config/spring-cassandra-1.0.xsd

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shoot, I did not realize that; darn it.  This is both unfortunate and redundant, and `spring-cql-1.0.xsd` should probably be **deprecated** since it is a subset of the `spring-cassandra-1.0` XSD.

Updated.